app/views/layouts/govuk_admin_template.html.erb

```diff
@@ -59,20 +60,28 @@
                 <span class="icon-bar"></span>
               </a>
             <% end %>
-            <%= link_to content_for?(:app_title) ? yield(:app_title) : "GOV.UK", app_home_path, :class => 'navbar-brand' %>
+            <%= link_to app_title, app_home_path, :class => 'navbar-brand' %>
             <% if environment_label %>
               <div class="environment-label">
                 <%= environment_label %>
               </div>
             <% end %>
           </div>
-          <% if content_for?(:navbar_right) || content_for?(:navbar_items) %>
+          <% if GovukAdminTemplate::Config.show_signout || content_for?(:navbar_right) || content_for?(:navbar_items) %>
```

Comment:
Yeah that's good. I'll do that or remove the conditionals entirely, in a later PR. Is that okay?